CHANGELOG.md

```diff
@@ -4,11 +4,14 @@ All notable changes to this project will be documented in this file based on the
 ## [Unreleased](https://github.com/ruflin/Elastica/compare/3.1.1...HEAD)
 
 ### Backward Compatibility Breaks
+- Method \Elastica\ResultSet::create and property \Elastica\ResultSet::$class were removed. To change the ResultSet class, implement your own ResultSet Builder. #1065
```

Comment:
The notes here except for the processing line were part of the previous PR, github obviously didnt update the diffs when it was merged.